schema/defs-linux.json

```diff
@@ -203,7 +204,8 @@
                     "$ref": "#/definitions/Minor"
                 },
                 "access": {
-                    "type": "string"
+                    "type": "string",
+                    "pattern": "^[rwm]{1,3}$"
```

Comment:
This regexp would allow `rrr`, which should be invalid.  But it's probably close enough, since a long chain of `|`ed options would be tedious to write, and I can't think of a better way to test for `r?w?m?` which is order-agnostic and includes at least one character.